src/github/pullRequestOverview.ts

```diff
@@ -672,37 +672,30 @@ export class PullRequestOverviewPanel {
 		});
 	}
 
-	private async checkoutDefaultBranch(branch: string): Promise<void> {
+	private async checkoutDefaultBranch(message: IRequestMessage<string>): Promise<void> {
 		try {
+			const branch = message.args;
 			// This should be updated for multi-root support and consume the git extension API if possible
 			const branchObj = await this._pullRequestManager.repository.getBranch('@{-1}');
 
-			if (branch === branchObj.name) {
+			if (branchObj.upstream && branch === branchObj.upstream.name) {
```

Comment:
`branchObj.name` seems to always be `@{-1}`. I can't remember what our original intention was with this logic, but this statement was previously always false